torch/csrc/Exceptions.h

```diff
@@ -325,6 +331,18 @@ struct AttributeError : public PyTorchError {
   }
 };
 
+namespace linalg {
```

Comment:
Not sure the name space is strictly needed here.
Also this should move next to THPException_FatalError